src/Internal/Payment/PaymentContext.php

```diff
@@ -244,17 +256,50 @@ public function get_intent(): ?WC_Payments_API_Abstract_Intention {
 		return $this->get( 'intent' );
 	}
 
+	/**
+	 * Returns the transitions array.
+	 *
+	 * @return array
```

Comment:
This could be defined as `Transition[]`.

Reply:
Done